soccer/Robot.cpp

```diff
@@ -630,6 +629,7 @@ void OurRobot::replanIfNeeded(const Geometry2d::CompositeShape& global_obstacles
 		//  TODO: This is Stupid. This should be fixed in the RRT planner or the Bezier Algorithm.
 		if (_path->destination() && (*_path->destination() - dest).mag() > 0.025) {
 			_pathInvalidated = true;
+			addText("Destination Moved");
```

Comment:
Sounds good, it's just kinda crufty when we have a bunch of text on our nice pretty soccer view :).  I do agree though that we should have a better way to see debug text like this when we want to.  Right now, when you call addText("my info"), it puts it in the "RobotText" layer.  Maybe we could put these planner-related things in a "RobotPlanningDebugText" layer and make it invisible by default, so if you want to see it, you could check the box in the layers tab.  Thoughts?  That would probably be best as a separate pull request though, this one is already pretty massive haha.